### DISCUSSION TITLE
Welcome to Sorting-Searching-Algorithms Discussions!

### DISCUSSION THREAD

**Discussion by 4N1Z** [4N1Z/Sorting-Searching-Algorithms#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AnirudhDaya** [comment]
We require some readme files for each of the searching and sorting algorithms, so open an issue with the name of the algorithm and write a proper explanation of the algorithm: 
1. How it works?
2. Maybe some references where you can learn more about it
Looking forward towards your issues and PRs. 😊

@4N1Z  is this much content enough for the readme?

**Comment by AnirudhDaya** [comment]
## Important Announcement!!!
Guys, we merge the PRs on a first-come-first-serve basis. So if there are two PRs of the same kind. The correct PR will be merged. So if we request changes, the one who does it quickly and correctly will be merged. So try and do it correctly while making a PR. Also watch out for duplicate ones.

**Comment by PranjalSK03** [comment]
can i add a new sorting algorithm that is not present in the list already?

**Comment by HarshAggarwal1** [comment]
Hi, @4N1Z. I have created a pull request for the java code of BOGO sort; please view that, and if it seems nice, please accept my request; can I add other sorting algorithms too, which are not mentioned in the list?
